packages/core/content-releases/server/src/__tests__/release.test.ts

```diff
@@ -0,0 +1,20 @@
+import releaseController from "../controllers/release";
+
+describe('Release controllers', () => {
+  describe('create', () => {
```

Comment:
I don't think so, at least here is not the place in my opinion.

Because permissions are defined in a route.ts file which is already tested and we know it's working, we are not adding (at least for now), any extra permission check on our controllers/services.

With the previous focus to check if user was a superadmin to perform an action makes sense to check that, but now everything is happening outside content-releases plugin

Reply:
ok I guess my concern is that potentially someone could remove the policy from the route on accident and we have no test case to ensure it is there.

Reply:
I think that's the kind of stuff we can write down to make sure we test it on the e2e tests 🤔

Reply:
sounds good